packages/nextjs/components/Card.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from "next/link";
+import Arrow_Icon from "~~/components/Arrow_Icon";
+
+type Props = {
+  mainText: string;
+  footerText: string;
+  footerLink: string;
+};
+
+const Card = ({ mainText, footerText, footerLink }: Props) => {
+  return (
+    <div className="shadow-sm border rounded-xl w-[16rem] h-[12rem] font-light mt-6 border-zinc-500 flex flex-col">
+      <div className="w-full flex items-center justify-center flex-grow-[5] dark:hover:bg-zinc-800 px-4 hover:bg-zinc-300 transition-all rounded-t-xl">
+        {mainText}
+      </div>
```

Comment:
Thanks for creating the card component! 

When I see a hover effect, it makes me want to click on it, but since there's no link on the upper part, you might want to either remove the hover effect or merge the upper and lower parts together.

Reply:
alright i'll remove it